rasa/core/policies/ted_policy.py

```diff
@@ -989,6 +1042,44 @@ def _encode_real_features_per_attribute(
 
             if attribute == TEXT:
                 text_transformer_output = attribute_features
+                text_sequence_lengths = sequence_lengths
+
+                if self.use_only_last_dialogue_turn:
+                    # get the location of all last dialogue inputs
+                    dialogue_lengths = tf.cast(
+                        tf_batch_data[DIALOGUE][LENGTH][0], dtype=tf.int32
+                    )
+                    # TODO precompute dialogue_indices after creation of tf_batch_data
```

Comment:
should we quickly take care of this TODO?

Reply:
yes, should be done as part of this PR